.github/workflows/release.dsg.template.yml

```diff
@@ -0,0 +1,123 @@
+            echo "new-tag=$NEW_TAG" >> $GITHUB_OUTPUT
+            echo "new-dsg-version=$DSG_VERSION" >> $GITHUB_OUTPUT
+
+            echo "🚀 New version detected"
+            echo " 🏷️ data-service-generator version: $DSG_VERSION"
+            echo " 🏷️ NEW_TAG: $NEW_TAG"
+            echo " 🏷️ PREVIOUS_TAG: $PREVIOUS_TAG "
+          fi
+          echo "is-new-version=$IS_NEW_VERSION" >> $GITHUB_OUTPUT
+        env:
+          DSG_PACKAGE_JSON: packages/data-service-generator/package.json
+
+      - name: Build data-service-generator Changelog
+        id: changelog
+        if: ${{ steps.check-tag.outputs.is-new-version == 'true' }}
+        uses: mikepenz/release-changelog-builder-action@v4
+        with:
+          fromTag: "${{ steps.check-tag.outputs.previous-tag }}"
+          toTag: HEAD # from actions/checkout@v3
+          configurationJson: |
+            {
+              "ignore_labels": [
+                "ignore-from-release-notes"
+              ],
+              "template": "#{{CHANGELOG}}\n\n**Full Changelog**: [#{{FROM_TAG}}...${{ steps.check-tag.outputs.new-tag }}](https://github.com/amplication/amplication/compare/#{{FROM_TAG}}...${{ steps.check-tag.outputs.new-tag }})",
+              "pr_template": "- #{{TITLE}} by @#{{AUTHOR}} in [##{{NUMBER}}](#{{URL}})",
+              "categories": [
+                {
+                  "title": "## [`data-service-generator`] Changes 🛠",
+                  "exhaustive_rules": "true",
+                  "rules": [
+                    {
+                      "pattern": "app:data-service-generator",
+                      "on_property": "labels",
+                      "flags": "gu"
+                    }
+                  ]
+                }
+              ]
+            }
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create data-service-generator git tag
+        if: ${{ steps.check-tag.outputs.is-new-version == 'true' }}
+        run: |
+          NEW_TAG="${{ steps.check-tag.outputs.new-tag }}"
+          git push origin $NEW_TAG
+
+      - name: Generate release notes artifact
+        if: ${{ steps.check-tag.outputs.is-new-version == 'true' }}
+        run: |
+          echo "${{ steps.changelog.outputs.changelog }}" >> release-notes.md
+
+      - name: Upload release notes artifact
+        if: ${{ steps.check-tag.outputs.is-new-version == 'true' }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: release-notes
+          path: release-notes.md
+
+      # - name: Release data-service-generator
+      #   id: release
+      #   if: ${{ steps.check-tag.outputs.is-new-version == 'true' }}
+      #   uses: softprops/action-gh-release@v1
+      #   with:
+      #     name: "[data-service-generator] ${{ steps.check-tag.outputs.new-dsg-version }}"
+      #     tag_name: "${{ steps.check-tag.outputs.new-tag }}"
+      #     draft: true
+      #     body: ${{ steps.changelog.outputs.changelog }}
```

Comment:
Is this supposed to be commented out?

Reply:
I'll remove it as we won't use this approach

Reply:
Solved